### PR TITLE
Stacked PR: Various rubocop autocorrections

### PR DESCRIPTION
**Pull request by CloCkWeRX**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -103,11 +103,12 @@ Lint/SymbolConversion:
   Exclude:
     - 'app/controllers/likes_controller.rb'
 
-# Offense count: 7
+# Offense count: 8
```
Comment (Br3nda) [inline .rubocop_todo.yml:106]: This count goes up

Comment (CloCkWeRX) [reply .rubocop_todo.yml:106]: True. What happens often is one set of exclusions is exempting a whole file. Then it gets fixed, and three other problems are found.

For now, am focused on auto correcting the safe changes - will chip away - and then regenerating the todo file to reflect current state regularly.

**Comment by cesy** [comment]
If you want to update this onto latest dev, I'm happy to do a final review and get some of it merged.

File: lib/actions/oauth_signup_action.rb [hunk 2]
```diff
@@ -63,13 +63,11 @@ def determine_name(auth)
     name = ''
```
Comment (cesy) [inline lib/actions/oauth_signup_action.rb]: Is this line still needed if we're taking out the rest of the definition?

**Comment by cesy** [comment]
All the other changes look good to me, so happy to merge apart from that one question.

**Review by cesy — APPROVED** [review]

File: lib/actions/oauth_signup_action.rb (hunk 2 above)
Comment (Br3nda) [reply lib/actions/oauth_signup_action.rb]: Surprised the linter allows that unused variable. Must be used somewhere? If so, maybe its important to set it?

Comment (Br3nda) [reply lib/actions/oauth_signup_action.rb]: __
```suggestion
```

**Review by Br3nda — APPROVED** [review]

**Pull request merged**